tck/README.md

```diff
@@ -479,6 +487,22 @@ public class SkippingIdentityProcessorTest extends IdentityProcessorVerification
 }
 ```
 
+## Which verifications must be implemented by a compliant implementation?
+In order to be considered an Reactive Streams compliant require implementations to cover their
```

Comment:
It was missing word (reactive streams compliant implementation), but that leads to repetition.
So I'll go with `In order to be considered Reactive Streams compliant, implementations must cover their...`